### PR TITLE
feat: verbatimModuleSyntax

### PR DESCRIPTION
**Pull request by farreldarian**

### CODE REVIEW

File: packages/generator/src/lib/syntaxes/imports.ts [hunk 1]
```diff
@@ -1,24 +1,36 @@
...
+) {
 	return {
 		type: 'namedImport' as const,
 		names: names,
 		module: path,
+		isTypeImport,
 		render() {
-			return `import { ${names.join(', ')} } from '${renderImportPath(path)}';`
+			// biome-ignore format: off
+			return `import ${isTypeImport ? 'type' : ''} { ${names.join( ', ')} } from '${renderImportPath(path)}';`
```
Comment (jansedlon) [inline packages/generator/src/lib/syntaxes/imports.ts:15]:
```suggestion
			return `import ${isTypeImport ? 'type ' : ''}{ ${names.join( ', ')} } from '${renderImportPath(path)}';`
```

File: packages/generator/src/lib/syntaxes/imports.ts [hunk 2]
```diff
@@ -1,24 +1,36 @@
...
+) {
 	return {
 		type: 'defaultImport' as const,
 		name,
 		module: path,
+		isTypeImport,
 		render() {
-			return `import ${name} from '${renderImportPath(path)}';`
+			// biome-ignore format: off
+			return `import ${isTypeImport ? 'type' : ''} ${name} from '${renderImportPath(path)}';`
```
Comment (jansedlon) [inline packages/generator/src/lib/syntaxes/imports.ts:33]:
```suggestion
			return `import ${isTypeImport ? 'type ' : ''}${name} from '${renderImportPath(path)}';`
```

**Review by jansedlon — CHANGES_REQUESTED** [review]

**Comment by jansedlon** [comment]
Just the two small changed, otherwise LGTM. Without the changes, there are 2 spaces.
Eg.

```ts
import  { pgEnum } from 'drizzle-orm/pg-core';
//         ||
```

**Comment by farreldarian** [comment]
> Btw i added the `docker-compose.yaml` purposefully so that any other person who would like to contribute had easier time to setup dev environment.

Added

**Pull request merged**